### PR TITLE
Feature/heh alternate exchanges

### PR DESCRIPTION
**Pull request by hansehe**
---
Rebus is [MIT-licensed](https://opensource.org/licenses/MIT). The code submitted in this pull request needs to carry the MIT license too. By leaving this text in, __I hereby acknowledge that the code submitted in the pull request has the MIT license and can be merged with the Rebus codebase__.

### CODE REVIEW

**Comment by hansehe** [comment]
PR for issue: https://github.com/rebus-org/Rebus.RabbitMq/issues/33

File: Rebus.RabbitMq.Tests/RabbitMqCustomExchangeNamesTest.cs [hunk 1]
```diff
@@ -71,14 +70,14 @@ public async Task CanUseAlternateCustomExchangeName()
             var topicWithAlternateExchange = $"{topic}@{alternateExchange}";
 
             var subscriberAddresses = await rabbitMqTransport.GetSubscriberAddresses(topicWithAlternateExchange);
-            subscriberAddresses[0].Should().Be(topicWithAlternateExchange);
+            Assert.IsTrue(subscriberAddresses[0] == topicWithAlternateExchange);
```
[the remark this answers is not in the document]
Comment (hansehe) [reply Rebus.RabbitMq.Tests/RabbitMqCustomExchangeNamesTest.cs]: thanks! I'll do the update :)

**Comment by hansehe** [comment]
Seems like the integration test failed due to a fail with building the rabbitMq?

**Comment by mookid8000** [comment]
Excellent work 👍 – one thing though: What was the reasoning behind "publish on alternate exchanges enabled"? Why not just say that that's the way it always works?

**Comment by mookid8000** [comment]
it's out in Rebus.RabbitMq 5.0.0-b08 now 😄

**Comment by mookid8000** [comment]
I'll remove it. I don't like too many options 😄 Version 5 is not out yet, so this is a good time to make this change.

**Comment by mookid8000** [comment]
it's removed in Rebus.RabbitMq 5.0.0-b09

**Pull request merged**